packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -80,6 +94,10 @@ const AuthDefault = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [shouldLogin]);
 
+  useLayoutEffect(() => {
+    focusFirstSocialLink();
+  }, []);
```

Comment:
@rebelchris I remember one session we had with tkDodo about use effects. Is it maybe possible to have this within the `ref` callback of the `div` element itself? I think so it should.

Reply:
Yeah should be possible indeed